data/src/main/java/pl/edu/agh/model/Sprint.java

```diff
@@ -1,5 +1,47 @@
 package pl.edu.agh.model;
 
+import java.time.Duration;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
 public class Sprint {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(nullable = false)
+    private Long id;
+
+    // TODO: hook up reference
+    @Transient
+    private Project project;
+
+    private LocalDateTimeRange date;
+    private LocalDateTimeRange enrollmentDate;
```

Comment:
W tych nazwach bym uwzględniła jakoś słowo "range". Nie wiem - `dateRange` i ` enrollmentDateRange`?

Reply:
może period, enrollmentPeriod?

Reply:
`timeRange` i `enrollmentTimeRange` jak dla mnie spoko

`beginning` :+1:

Reply:
nie jestem przekonany, chcialbym zeby ktos jeszcze sie wypowiedzial tutaj

Reply:
@Sopel97 rzeczywiście nazwa `date` nie sugeruje, że to struktura reprezentująca pewien przedział czasowy. Wydaje mi się, że powinniśmy być konsekwentni, jeśli nazywamy klasę `*Range` to pole też, jeśli `*Period` to pole również. Osobiście lepiej mi pasuje `period` niż `range`, bo [period](https://en.wiktionary.org/wiki/period) odnosi się bezpośrednio do czasu, a [range](https://en.wiktionary.org/wiki/range) kojarzy się często z wartościami dyskretnymi. Również starałbym się używać nazw `time` zamiast `date`, bo przechowujemy to z dokładnością do czasu, a nie daty.

Reply:
`timePeriod` dla mnie też może być, ale nie zostawiałabym samego `period` na pewno - i wtedy faktycznie klasa pewnie też